model/scenarios/30-mor-fødsel-AAP-SIGRUN/inntektytelse-søker.json

```diff
@@ -0,0 +1,95 @@
+{
+	"arena": {
+		"saker": [
+			{
+				"saksnummer": "${arenasak1}",
+				"tema": "AAP",
+				"status": "AKTIV",
+				"vedtak": [
+					{
+						"fom": "now() - P1M",
+						"tom": "now() + P12W",
+						"vedtakDato": "now() - P12M",
+						"kravMottattDato": "now() - P12M",
+						"status": "IVERK",
+						"dagsats": 1000,
+						"meldekort": [
+							{
+								"fom": "now() - P1M",
+								"tom": "now() + P12W",
+								"dagsats": 1000,
+								"beløp": 10000,
+								"utbetalingsgrad": 200
+							}
+						]
+					}
+				]
+			}
+		]
+	},
+	"inntektskomponent": {
```

Comment:
Tomt element - Kan dette fjernes eller får vi feil da? (Isåfall kan vi lage en task på å gjøre feltet valgfritt.